contrib/nydusify/pkg/viewer/viewer.go

```diff
@@ -167,6 +179,18 @@ func (fsViewer *FsViewer) View(ctx context.Context) error {
 		return errors.Wrap(err, "failed to pull Nydus image bootstrap")
 	}
 
+	//Adjust nydused parameters(DigestValidate) according to rafs format
+	nydusManifest := parser.FindNydusBootstrapDesc(&targetParsed.NydusImage.Manifest)
```

Comment:
It looks unnecessary.

Reply:
The parameter 'nydusManifest' is used to get the version of fs as fallows  (the third line)


```
nydusManifest := parser.FindNydusBootstrapDesc(&targetParsed.NydusImage.Manifest)
if nydusManifest != nil {
v := utils.GetNydusFsVersionOrDefault(nydusManifest.Annotations, utils.V5)//this place
if v == utils.V5 {
	// Digest validate is not currently supported for v6,
	// but v5 supports it. In order to make the check more sufficient,
	// this validate needs to be turned on for v5.
	fsViewer.NydusdConfig.DigestValidate = true
}
```